_source/_posts/2018-02-01-secure-aspnetcore-webapi-token-auth.md

```diff
@@ -343,15 +345,20 @@ namespace app.Services
       {
         get
         {
-          return !String.IsNullOrEmpty(this.AccessToken) && t
-his.ExpiresAt > DateTime.UtcNow.AddSeconds(30);
+          return !String.IsNullOrEmpty(this.AccessToken) && this.ExpiresAt > DateTime.UtcNow.AddSeconds(30);
         }
       }
     }
   }
 }
 ```
 
+Install [Json.NET](https://www.newtonsoft.com/json) using the following command:
+
+```
+dotnet add package Newtonsoft.Json --version 13.0.1
+```
```

Comment:
@leebrandt I'm not sure this is needed, but it was for .NET 5. Maybe it's built-in for .NET 2?